code/style.css

```diff
@@ -147,3 +148,17 @@ button:hover {
   opacity: 0.9;
   transition: all 0.2s ease;
 }
+
+select {
+  box-sizing: border-box;
+  border: none;
+  border-radius: 4px 0 0 4px;
+  background: #e5e9ff;
+  color: #0026ff;
+  padding: 16px;
+  font-size: 16px;
+  font-family: 'Montserrat';
+  font-weight: 600;
+  line-height: 26px;
+  flex: 1;
+}
```

Comment:
For your portfolios sake, should you maybe put some time into making it responsive with an media query for the messages (up the % on the bubbles) and maybe change the sizes of the buttons and input and set the main to another height and width (maybe less border-radius?)... this is a fast fix, you're a css code pro! 💪🏼